### DISCUSSION TITLE
Drop-in replacement for dohq-artifactory?

### DISCUSSION THREAD

**Discussion by okainov** [anancarv/python-artifactory#189] [Q&A] [answered]
Seems like this project is quite active in contrast with `dohq-artifactory` https://github.com/devopshq/artifactory. Given that they're not compatible with Python 3.12, one might consider switching.

I haven't done any research myself at the moment, but how complicated it would be to replace? If not "drop-in" replacement, but at least with minimal changes?

**Comment by anancarv** [comment, marked as answer]
Hey @okainov,
it shouldn't be that complicated to switch to this library. Unfortunately, there is no migration guide, but I suggest reading through the documentation for examples matching your use cases. This will give you insights into how to perform the same tasks with `pyartifactory`. It should be pretty straightforward
